src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -1,34 +1,45 @@
 package tech.cryptonomic.conseil
 
 import akka.actor.ActorSystem
-import akka.Done
-import mouse.any._
 import com.typesafe.scalalogging.LazyLogging
 import tech.cryptonomic.conseil.tezos.{
-  TezosTypes,
+  AccountsDataFetchers,
+  ApiOperations,
+  BlocksDataFetchers,
   FeeOperations,
-  ShutdownComplete,
+  NodeOperator,
+  TezosDatabaseOperations => TezosDb,
   TezosErrors,
-  TezosNodeInterface,
-  TezosNodeOperator,
-  TezosDatabaseOperations => TezosDb
+  TezosTypes
 }
-import tech.cryptonomic.conseil.tezos.TezosTypes.{Account, AccountId, BlockTagged, Delegate, PublicKeyHash}
+import tech.cryptonomic.conseil.tezos.TezosRemoteInstances.Akka.{ShutdownComplete, TezosNodeContext}
+import tech.cryptonomic.conseil.tezos.TezosTypes._
 import tech.cryptonomic.conseil.io.MainOutputs.LorreOutput
 import tech.cryptonomic.conseil.util.DatabaseUtil
+import tech.cryptonomic.conseil.util.EffectsUtil.{runDbIO, toIO}
 import tech.cryptonomic.conseil.config.{Custom, Everything, LorreAppConfig, Newest}
 import tech.cryptonomic.conseil.config.Platforms
 
 import scala.concurrent.duration._
 import scala.annotation.tailrec
-import scala.concurrent.{Await, Future}
+import scala.concurrent.Await
 import scala.concurrent.duration.Duration
 import scala.util.{Failure, Success, Try}
+import scala.{Stream => _}
```

Comment:
I'm just guessing what id does - it disables predefined scala `Stream` so you could use `fs2` without any issues, am I right?

Reply:
Absolutely... to avoid unwanted conflicts/confusion